### ISSUE TITLE
'NoneType' object has no attribute 'get'

### ISSUE THREAD

**Issue by sipi1986** [shr00mie/gmapslocsharing#4]
Running Home Assistant version 0.98.1 on docker and getting this error in the Home Assistant logs: "Google Maps - Component configuration failed: 'NoneType' object has no attribute 'get'.".

I set debug to true but unfortunately there is no extra information in the debug folder. 
Any idea what could be the issue? Maybe Home Assistant version 0.98.1 is not supported yet (I haven't tried this component in earlier versions of Home Assistant)?

**Comment by shr00mie** [comment]
Ah. Yeah. I think 98 is the first version with the new Alpine image. The current docker config is for the old python image based on I think Debian or Ubuntu. Going to have to reconfigure for Alpine this week when I get back from vacation on Wednesday.

**Comment by shr00mie** [comment]
almost there. doing some polishing. i'll probably push something today.

![image](https://user-images.githubusercontent.com/7096317/64261190-faf70680-cee0-11e9-9fd9-433b07d7e5ee.png)

**Comment by sipi1986** [comment]
Looking good, thanks for the effort!

**Comment by shr00mie** [comment]
@sipi1986 alright. pushed to docker branch. give it a go and let me know if there are (probably) any some lingering issues.

**Comment by sipi1986** [comment]
Definately got some steps further, the original error disappeared but now I get:
ERROR (SyncWorker_2) [gmapslocsharing.core.browser] Error entering email: malformed node or string: <_ast.BinOp object at 0x7ff29ef6dcd0>.
ERROR (SyncWorker_2) [gmapslocsharing.core.browser] Error entering password: malformed node or string: <_ast.BinOp object at 0x7ff29ef6de50>.
This seems to occur in email.send_keys(self.config.email) and password.send_keys(self.config.password).
Not sure if this error is related to the switch to Alpine or maybe just related to me doing something wrong. 

I also get output in the debug folder containing screenshots and urls. Unfortunately nothing that really gets me any further regarding the error.

**Comment by shr00mie** [comment]
what did you update after my last push? I also made some changes to the platform files. specifically config get method.

would it be safe to assume that you are using special characters in your password, and they may include, but not be limited to '@#$=&:'?

**Comment by sipi1986** [comment]
Correct in that my password contained special characters. I changed my password to something without special characters and also removed the @gmail.com from the username and now get this:
ERROR (SyncWorker_8) [gmapslocsharing.core.browser] Error entering email: local variable 'c' referenced before assignment.
ERROR (SyncWorker_8) [gmapslocsharing.core.browser] Error entering password: local variable 'c' referenced before assignment.

I updated everything in the custom component en deps folders and ran the scripts from the dockerfile
